examples/user_guide/Lazy_Data.ipynb

```diff
@@ -0,0 +1,107 @@
+    "In addition, HoloViews can also accept Ibis objects, and can plot data making use of the database to actually compute the expressions stored in the lazy object task graph.  The syntax to create HoloViews plots will be the same whether using a Pandas, Dask, or Ibis object.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Persist vs Compute\n",
+    "\n",
+    "When using lazy evaluation of data, there are 2 different calls you can make on the lazy data objects--`.compute()` or `.persist()`.  The call to `.compute()` is just like it sounds, this is the call to actually run the computation stored in your lazy object.  This function will compute the result and instead of a task graph, you will get the actual computation returned to you.  A Dask (or Ibis) dataframe computed would result in a single Pandas dataframe on your local computer. This should only be done if the result can fit into memory.\n",
+    "\n",
+    "In some cases, it may be useful to obtain results mid-way through a computation, but leave the results split up among the different parallel processes that make up the entire computation.  If you have a Dask cluster that is running a computation on several partitions of a large Dask dataframe, you can use `.persist()` to compute something on each of the partitions in the cluster.  The result of each of those partitions will be stored on their respective nodes.  The object returned to you will now point to those running processes, where the computation is stored in memory.  The object you see is still a lazy object, but some part of the computation is persisting in memory split among many nodes.  The benefit of this is that you do not have to keep running the same computation over and over again every time you need to do a different computation or plot further down the task chain."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### When would you want to use .persist() when using HoloViews?\n",
+    "\n",
+    "If you are going to be doing any computations before plotting (aggregating or reducing your data, for example), calling .persist() on these computations will avoid evaluating that portion of the task graph several times.  These computations will be stored in memory on the cluster nodes, ready to be used when needed for your various plots.  \n",
+    "\n",
+    "### When do you use .compute()?  \n",
+    "\n",
+    "Some methods you may use will not support data being stored on multiple nodes.  You may need to perform a sort on the entire dataset or get an index location with with `.iloc()`, and these require putting all the data on one node.  In this case, it will be best to do as much as you can to reduce or aggregate your data on each node, and then use `.compute()` to bring everything to a single node.  You can then use the methods that do not work across nodes to obtain your desired result.  "
```

Comment:
I'd say something about ideally `.compute()` would return a reduced version of the original data -- aggregated, sliced, selected, etc., so that the resulting object is reasonable to fit in a single machine's memory even for a large distributed dataset.